### PR TITLE
fix issue on content type reload

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Resources/public/js/backboneRouter.js [hunk 1]
```diff
@@ -166,20 +166,17 @@ var OrchestraBORouter = Backbone.Router.extend({
 
   generateUrl: function(routeName, paramsObject)
   {
+    var optionalParam = /\(([^)]*):([^)]*)\)/g;
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/js/backboneRouter.js]: you should precise the _ in the regex

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/js/backboneRouter.js]: Why?

Comment (nicolasThal) [inline BackofficeBundle/Resources/public/js/backboneRouter.js]: So your regex would be more  linked to the pattern you have put `/\(([^)]*_):([^)]*)\)/g`

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/js/backboneRouter.js]: the pattern is an arbitrary choice. Could be language.fr or languagefr or anything else. But in order to have no conflict, I add '_'. Here we are in remplacement process, so separator character has no importance. We could choose to change separator in route, if we got conflict (i.e. language_fr is an entity) without change resolve routing process. @nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**